packages/design-tokens/src/js/tokens.ts

```diff
@@ -224,45 +224,49 @@ export const tokens: TokenStructure = {
       letterSpacing: "0em",
     },
     heading1: {
```

Comment:
Potentially worth updating the composable header variant in Heading to use these :)